include/tvm/relay/expr.h

```diff
@@ -230,10 +233,14 @@ class FunctionNode : public ExprNode {
    */
   TVM_DLL FuncType func_type_annotation() const;
 
+  TVM_DLL NodeRef GetAttr(const std::string& key) const;
```

Comment:
remove the setters

Reply:
?

Reply:
As per discussion, we want to keep attrs immutable, and thus these setters are no longer valid

Reply:
They don't mutate; they build an identical function with the attributes updated and hide the representation in case we modify it in the future.